src/hooks/custom-hooks/__tests__/useTrackjs.tsx

```diff
@@ -0,0 +1,42 @@
+import { TrackJS } from 'trackjs';
```

Comment:
why is the test file useTrackjs.tsx ? 🤔

Reply:
good catch 😄 I'll rename it. thanks @nada-deriv